data-serving/data-service/api/openapi.yaml

```diff
@@ -132,6 +151,12 @@ components:
       description: The subset of "#/components/schemas/Case" required to create the entity
       type: object
   responses:
+    "200Case":
```

Comment:
Worth adding a 201Case, just so these are all in the same place?

Reply:
SG, made a 200CaseArray and moved it there as well